src/pages/About/AboutKyberSwap/index.tsx

```diff
@@ -87,7 +87,8 @@ import {
   TypicalAMM,
   VerticalDivider,
   Wrapper,
-} from './styleds'
+} from '../styleds'
```

Comment:
Better to not using `../`. Use absolute link import instead.

Reply:
Resolve as I explained above